app/backend/src/couchers/models.py

```diff
@@ -117,6 +117,8 @@ class User(Base):
 
     # id of the last message that they received a notification about
     last_notified_message_id = Column(BigInteger, nullable=False, default=0)
+    # same as above for host requests
+    last_notified_request_message_id = Column(BigInteger, nullable=False, server_default=text("0"))
```

Comment:
```suggestion
    last_notified_request_message_id = Column(BigInteger, nullable=False, server_default=0)
```

Reply:
SQLAlchemy won't take that... It needs to be an SQL text fragment! It's pretty dumb... I know...

Reply:
I saw 0 on the line above, but that's not a server_default. Are you saying we should change all default to server_default?

Reply:
LOL, "there's an [app](https://sqlalchemy-defaults.readthedocs.io/en/latest/) for that"

Reply:
> LOL, "there's an [app](https://sqlalchemy-defaults.readthedocs.io/en/latest/) for that"

🤦

Reply:
> I saw 0 on the line above, but that's not a server_default. Are you saying we should change all default to server_default?

Yeah, we probably should... Depending on it, `default` is python side, `server_default` is postgres side.

Probably most should be `server_default`s, but it's just, yeah, this text stuff is silly